test/e2e/flask/btc/btc-account-overview.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { strict as assert } from 'assert';
+import { Suite } from 'mocha';
+import {
+  withFixtures,
+  defaultGanacheOptions,
+  unlockWallet,
+} from '../../helpers';
+import { Driver } from '../../webdriver/driver';
+import FixtureBuilder from '../../fixture-builder';
+import { createBtcAccount } from '../../accounts/common';
+
+describe('BTC Account - Overview', function (this: Suite) {
+  it('has buy/sell and portfolio button enabled for BTC accounts', async function () {
```

Comment:
For now, only that one is enabled:

```suggestion
  it('has portfolio button enabled for BTC accounts', async function () {
```